### ISSUE TITLE
### <span aria-hidden="true">❌</span> Deploy Preview for *evm-chainlist* failed.

### ISSUE THREAD

**Issue by jraah555** [FrederikBolding/chainlist#27]
### <span aria-hidden="true">❌</span> Deploy Preview for *evm-chainlist* failed.

|  Name | Link |
|---------------------------------|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit |  |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/sites/evm-chainlist/deploys/63371adfcda79900466ac565 |

_Originally posted by @netlify[bot] in https://github.com/FrederikBolding/chainlist/issues/16#issuecomment-1263784310_
